source/scss/_library/_objects.sections.scss

```diff
@@ -209,3 +226,103 @@
   }
 
 }
+
+
+/**
+ * Section listing header
+ */
+.c-section__header {
+  padding: $space-double $space;
+  background-color: $c-quaternary;
+  text-align: center;
+  margin-bottom: $space-double;
+
+  @include media('>medium') {
+    padding: $space-triple $space;
+    margin-bottom: $space-quad;
+  }
+}
+.c-section__heading {
+  font-weight: normal;
+  font-size: $font-size-m;
+  letter-spacing: 3px;
+
+  @include media('>medium') {
+    font-size: $font-size-xl;
+  }
+}
+
+
+/**
+ * Tag listing header
+ */
+.c-tag-listing {
+
+  &__heading {
```

Comment:
see my other note about `header/heading`

Reply:
See my above explanation. If this isn't working for you, let's talk it through and try to come up with a workable solution.